app/root.tsx

```diff
@@ -13,7 +13,7 @@ export const links: LinksFunction = () => [{ rel: "stylesheet", href: styles }];
 
 export const meta: MetaFunction = () => ({
   charset: "utf-8",
-  title: "New Remix App",
+  title: "Matador",
```

Comment:
We should not set the title here, because this represent the title of the whole app, while Matador is just a template.

We could however set the title for the `/matador` route exporting a `MetaFunction`.

Reply:
Reverted with 2bcd18e27edbd9f8f70071dbee928ee3a9e613ec